mercury/models.py

```diff
@@ -124,3 +124,74 @@ class EventCodeAccess(models.Model):
 
     def __str__(self):  # pragma: no cover
         return EventCodeAccess.__name__
+
+
+class Events(models.Model):
```

Comment:
Yes, this module is not used right now. However, I think it could be merged in master, and be deleted in a soon followed pull request. This is because, first of all, this pull request is already too complicated. And we don’t want it to be bigger. Meanwhile, if we remove this, it will take the team extra time to verify the status of the pull request, as failures and checks can occur during tests and checks.

Reply:
sounds good